cloud-service-providers/aws/eks/nim-operator-setup.md

```diff
@@ -0,0 +1,142 @@
+# NVIDIA NIM Operator on AWS EKS:
```

Comment:
Can you add a link to this NIM Operator deployment in the main readme.md for easier discoverability?